configurations/es6-node-test.js

```diff
@@ -0,0 +1,19 @@
+"use strict";
+
+module.exports = {
+  "extends": [
+    "defaults/configurations/es6-node"
```

Comment:
That's the last PR is to fix all the places that say defaults. I didn't want to do that until this had gone in because it adds a new file that would have bad paths.